packages/hoppscotch-app/components/http/ResponseMeta.vue

```diff
@@ -141,6 +145,36 @@ const props = defineProps<{
   response: HoppRESTResponse
 }>()
 
+const responseSizeTippy = computed(() => {
+  if (
+    props.response.type === "loading" ||
+    props.response.type === "network_fail" ||
+    props.response.type === "script_fail" ||
+    props.response.type === "fail"
+  )
+    return
+  if (Number(props.response.meta.responseSize) <= 1000)
+    return {
+      onShow: () => {
+        return false
+      },
+    }
+  else return { theme: "tooltip", allowHTML: true }
+})
```

Comment:
Existence of `responseSizeTippy` is against the Vue ethos. This should be in the template and can be simplified if based against `responseSizeHint`.

```vue
<span
            v-if="response.meta && response.meta.responseSize"
            v-tippy="
              responseSizeHint
                ? { theme: 'tooltip', allowHTML: false }
                : { onShow: () => false }
            "
            :title="responseSizeHint"
          >
```